assets/css/v2/busway/departures.scss

```diff
@@ -1,3 +1,7 @@
 .departures-container {
   height: 100%;
+  padding-top: 0;
 }
+
+@import "departures/header";
+@import "departures/arrow";
```

Comment:
Hmmm 🤔. Moving these imports to the top of the file won't break anything so I'll go ahead and do that. However, the ordering and location of imports does impact the application of rules whose selectors have the same specificity. I'm not sure if we could move all existing not-at-the-top-of-the-file `@import`s to the top of the file and produce without introducing visual bugs.

We are probably best off asking for this in new code but leaving existing stuff as-is.

---

FWIW I was referencing the patterns established by `v2/lcd_common/departures.scss` when I did it this way:

https://github.com/mbta/screens/blob/ba872ea4d3e878122fc5b604a72877d436ba0f61/assets/css/v2/lcd_common/departures.scss#L1-L11

Reply:
> I'm not sure if we could move all existing not-at-the-top-of-the-file `@import`s to the top of the file and produce without introducing visual bugs.

Yes, I couldn't easily verify whether this was the case when I was setting up Stylelint, so I chose to disable the rule (since the problem it's meant to warn you about doesn't actually exist in SCSS anyway). But I did note this in the SCSS docs:

> ⚠️ **Heads up!**
>
> The Sass team discourages the continued use of the `@import` rule. Sass will [gradually phase it out](https://github.com/sass/sass/blob/main/accepted/module-system.md#timeline) over the next few years, and eventually remove it from the language entirely. Prefer the [`@use` rule](https://sass-lang.com/documentation/at-rules/use) instead.

Sass `@use` has the same restriction as vanilla CSS `@import`: it can appear only as the first thing in a file. So thinking ahead to a possible future where we migrate to `@use`, it's probably for the best if we pre-emptively follow this restriction and make sure our styles work correctly that way.

Reply:
Oh good to know! I tried to swap out the few uses of `@import` I added but ran into issues with referencing the `font--inter--bold` mixin I added (even after also including a `@use` for `fonts.scss` in the file using the mixing). 

Looks like there's a migration tool though so we can probably make the switch using that in another PR: https://sass-lang.com/documentation/cli/migrator/